lib/helpers/sassNodeModulesAliasImporter.js

```diff
@@ -0,0 +1,49 @@
+var path = require("path");
```

Comment:
You can use `const` (and `let`)

Reply:
Seemed to me that most of mango-cli is written with `var`s… but ok